evidence/pool.go

```diff
@@ -14,12 +14,17 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-// Pool maintains a pool of valid evidence in an Store.
+const (
+	baseKeyCommitted = byte(0x00) // committed evidence
```

Comment:
will changing these constants break anything? it looks like they were previously strings

Reply:
Yeah, so I am not to sure what degree of breaking they are but it will change how the evidence db is used i.e one will not be able to use the db from prior to this change to after this change. I briefly consulted Anton and I believe he said that it wouldn't be so much of a problem because the evidence db was seldom if not at all used

Reply:
hmm. it still sounds like we should probably consider this a breaking change. what do you think, @melekes?

Reply:
Also there will be more breaking changes once the amnesia evidence is sorted - we will need to add the functionality to add polka's to the evidence db

Reply:
yeah, it's breaking